x/group/README.md

```diff
@@ -330,13 +340,14 @@ The `voteTable` is a primary key table and its `PrimaryKey` is given by
 
 ### Msg/CreateGroup
 
-A new group can be created with the `MsgCreateGroup`, which has an admin address, a list of members and some optional metadata.
+A new group can be created with the `MsgCreateGroup`, which has an admin address, a list of members and some optional
+metadata.
 
 The metadata has a maximum length that is chosen by the app developer, and
 passed into the group keeper as a config.
 
 ```go reference
-https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/proto/cosmos/group/v1/tx.proto#L67-L80
+https: //github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc1/proto/cosmos/group/v1/tx.proto#L67-L80
```

Comment:
I think it was best to just fix the table manually.
All these changes will break the codeblock to be displayed on the website.